app/digitizingcontroller.h

```diff
@@ -56,8 +56,8 @@ class DigitizingController : public QObject
     Q_INVOKABLE FeatureLayerPair pointFeatureFromPoint( const QgsPoint &point, bool isGpsPoint );
     //! Creates a new QgsFeature with line/polygon geometry from the points stored since the start of recording
     Q_INVOKABLE FeatureLayerPair lineOrPolygonFeature();
-    //! Creates a new QgsFeature without geometry
-    Q_INVOKABLE FeatureLayerPair featureWithoutGeometry();
+    //! Creates a new QgsFeature without geometry in layer. If layer is null, it creates the feature on currently edited layer
```

Comment:
Thinking about to move this function to some generic/helper class since its used not only while digitizing, but also while creating linked feature. It has got a bit more custom I would say

Reply:
See comment below: https://github.com/lutraconsulting/input/pull/1531#discussion_r662282041